Snowplow.Analytics.Tests/Json/EventTransformerTest.cs

```diff
@@ -632,6 +632,200 @@ private string GetSerializedExpectedOutputForInputWithoutContextAndUnstructEvent
+                  'elementClasses' : [ 'foreground' ],
+                  'elementId' : 'exampleLink'
+                },
+                'tr_orderid' : null,
+                'tr_affiliation' : null,
+                'tr_total' : null,
+                'tr_tax' : null,
+                'tr_shipping' : null,
+                'tr_city' : null,
+                'tr_state' : null,
+                'tr_country' : null,
+                'ti_orderid' : null,
+                'ti_sku' : null,
+                'ti_name' : null,
+                'ti_category' : null,
+                'ti_price' : null,
+                'ti_quantity' : null,
+                'pp_xoffset_min' : null,
+                'pp_xoffset_max' : null,
+                'pp_yoffset_min' : null,
+                'pp_yoffset_max' : null,
+                'useragent' : 'Mozilla/5.0 (Linux; Android 9; SM-G950U Build/PPR1.180610.011; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/94.0.4606.61 Mobile Safari/537.36 (""Affirm - Android - 3.112.3 - 509\"")',
+                'br_name' : null,
+                'br_family' : null,
+                'br_version' : null,
+                'br_type' : null,
+                'br_renderengine' : null,
+                'br_lang' : null,
+                'br_features_pdf' : true,
+                'br_features_flash' : false,
+                'br_features_java' : null,
+                'br_features_director' : null,
+                'br_features_quicktime' : null,
+                'br_features_realplayer' : null,
+                'br_features_windowsmedia' : null,
+                'br_features_gears' : null,
+                'br_features_silverlight' : null,
+                'br_cookies' : null,
+                'br_colordepth' : null,
+                'br_viewwidth' : null,
+                'br_viewheight' : null,
+                'os_name' : null,
+                'os_family' : null,
+                'os_manufacturer' : null,
+                'os_timezone' : null,
+                'dvce_type' : null,
+                'dvce_ismobile' : null,
+                'dvce_screenwidth' : null,
+                'dvce_screenheight' : null,
+                'doc_charset' : null,
+                'doc_width' : null,
+                'doc_height' : null,
+                'tr_currency' : null,
+                'tr_total_base' : null,
+                'tr_tax_base' : null,
+                'tr_shipping_base' : null,
+                'ti_currency' : null,
+                'ti_price_base' : null,
+                'base_currency' : null,
+                'geo_timezone' : null,
+                'mkt_clickid' : null,
+                'mkt_network' : null,
+                'etl_tags' : null,
+                'dvce_sent_tstamp' : null,
+                'refr_domain_userid' : null,
+                'refr_device_tstamp' : null,
+                'contexts_com_snowplowanalytics_snowplow_ua_parser_context_1': [{
```

Comment:
nit: this has been copied from the example above, but wouldn't this context entity be generated from the provided useragent?

Reply:
This is a static value, it's not computed from the useragent string.